### PR TITLE
APIs separated

### LINKED ISSUE #24: Separate API to different apps
Use Blueprints attached to main Blueprint

### PR DESCRIPTION
**Pull request by Nikulin-V**
APIs have been separated between apps.
Socket and users API moved to area app.
Other APIs moved to market app.

### CODE REVIEW

File: market/templates/market/create-company.html [hunk 1]
```diff
@@ -4,12 +4,14 @@
 {% block content %}
 
     <script>
-        const imageUploadEndpoint = "{{ url('api.uploadImage') }}"
+        const imageUploadEndpoint = "{{ url('market.market-api.uploadImage') }}"
```
Comment (markK24) [inline market/templates/market/create-company.html]: 😬

**Review by markK24 — CHANGES_REQUESTED** [review]
I don't understand the purpose. It was better before, now the paths are too long, easier to get lost

**Comment by Nikulin-V** [comment]
Noticed a decrease in the speed of the site
Update: It is only when running throw Flask. If run throw Python, speed is good.

File: market/templates/market/create-company.html (hunk 1 above)
Comment (Nikulin-V) [reply market/templates/market/create-company.html]: Changed to "market.api.uploadImage"

**Review by markK24 — APPROVED** [review]

**Pull request merged**